sfputil/main.py

```diff
@@ -950,6 +1010,10 @@ def reset(port_name):
         click.echo("Error: No physical ports found for logical port '{}'".format(port_name))
         return
 
+    if is_rj45_port_from_api(port_name):
+        click.echo("Reset is not applicable for RJ45 port {}.".format(port_name))
```

Comment:
same. as the API platform already handle it why in sonic utilities we need again to differentiate?